bindings/c/test/mako/mako.c

```diff
@@ -1565,6 +1586,7 @@ void usage() {
 	printf("%-24s %s\n", "-v, --verbose", "Specify verbosity");
 	printf("%-24s %s\n", "-a, --api_version=API_VERSION", "Specify API_VERSION to use");
 	printf("%-24s %s\n", "-c, --cluster=FILE", "Specify FDB cluster file");
```

Comment:
Just to clarify, if you want to connect to 2 databases, you need to pass `-c` twice? Isn't it more intuitive to take a comma-separated list?